examples/cisco/demo_hsrp.pp

```diff
@@ -47,6 +56,12 @@
       hsrp_version       => 2,
     }
 
+    cisco_command_config { 'ipv6-addr':
```

Comment:
I could not find any property with which I can set ipv6 address in the cisco_interface provider. I do see ipv4 address but no ipv6. If I am missing something, please correct me and point me to the right property.